plugin/api.go

```diff
@@ -254,6 +254,11 @@ type API interface {
 	// Minimum server version: 5.6
 	GetProfileImage(userId string) ([]byte, *model.AppError)
 
+	// GetEmojiList returns a page of custom emoji on the system.
+	//
```

Comment:
Last additional suggestion would be to document the `sortBy` parameter, but otherwise LGTM!
```suggestion
	//
	// The sortBy parameter can be: "name".
	//
```